AmplifyPlugins/API/AWSAPICategoryPlugin/Core/AppSyncList.swift

```diff
@@ -10,6 +10,48 @@ import Amplify
 
 public class AppSyncList<ModelType: Model>: List<ModelType>, ModelListDecoder {
 
+    var storedElements: [Element]
+
+    public override
+    var elements: Elements {
+        storedElements
+    }
+
+    // MARK: - Initializers
+
+    init(_ elements: Elements) {
+        self.storedElements = elements
+        super.init()
+    }
+
+    // MARK: - ExpressibleByArrayLiteral
+
+    required convenience init(arrayLiteral elements: List<ModelType>.Element...) {
+        self.init(elements)
+    }
+
+    // MARK: Collection conformance
+
+    public override var startIndex: Index {
```

Comment:
I don't see that these are implicitly loading?

Reply:
going over the latest comments to apply to latest PR; from discussion with @drochetti, the observed behavior is that `makeIterator()` is called when startIndex is called, thus the "loadIfNeeded()" is in fact called. In the latest PR, we're performing the load when the computed property `elements` is accessed, so this will also be covered by the computed property logic